ql/src/test/queries/clientpositive/authorization_privilege_objects.q

```diff
@@ -0,0 +1,20 @@
+--! qt:authorizer
+set hive.security.authorization.manager=org.apache.hadoop.hive.ql.security.authorization.plugin.sqlstd.SQLStdHiveAuthorizerFactoryForTest;
+set test.hive.authz.sstd.validator.outputPrivObjs=true;
+set hive.test.authz.sstd.hs2.mode=true;
+set user.name=testuser;
+
+CREATE DATABASE test_db;
+CREATE TABLE test_privs(i int);
+set user.name=testuser2;
+CREATE TABLE test_privs2(s string, i int);
+set user.name=testuser;
+SHOW DATABASES;
+SHOW TABLES;
```

Comment:
nit: I'm wondering these queries doesn't  reach the `getTableMeta` method.

Reply:
I am unsure what you are trying to highlight.

The SHOW TABLEs might hit getTableMeta - however the purpose of this test isn't to specifically test getTableMeta, it is to show the actual HivePrivilegeObjects that end up getting passed to the authorization implementation(s). I couldn't find any other test that verified the HivePrivilegeObject being generated for various commands. I added a few other query types - SELECTs/INSERTs for example to add some coverage for them, but this test could be expanded to include many more statements (I just felt maybe trying to cover them all is a bit out of scope for this PR).

Reply:
I see, we also push owner info to authorization lawyer for `filter(List<Table>)`